docs/src/migration.md

```diff
@@ -80,17 +80,114 @@ we plan to support them soon.
 
 For JS/TS users, we offer a brand new SDK [@lancedb/lancedb](https://www.npmjs.com/package/@lancedb/lancedb)
 
+### Changes to DB APIs
+
+We tried to keep the API as similar as possible to the previous version, but there are a few small changes.
```

Comment:
We say there are a few small changes, but don't describe any of them. Could we do that?

Reply:
don't the following sub sections explain the differences?

- creating tables
- creating indices
- embedding functions

Reply:
Oh. I was confused because the next subheader is also an h3, so I didn't read that as a subsection. I guess `Creating tables` should be an h4 then?

Reply:
The hierarchy you have right now is

* TypeScript/JavaScript
  * Changes to DB APIs
  * Creating Tables
  * Changes to Table APIs
    * Creating Indices
  * Embedding Functions

Is that what you meant?

Is "Changes to DB APIs" meant to refer just to connection objects? Or to all the APIs? If the latter, I wonder if that header is even necessary.